Lib/test/test_httplib.py

```diff
@@ -1081,6 +1081,15 @@ def test_chunked(self):
         self.assertEqual(resp.read(), expected)
         resp.close()
 
+        # Explicit full read
+        for n in (-1, None):
```

Comment:
```suggestion
        for n in (-123, -1, None):
```

We can also try this because we want all negative values to read everything (sorry I forgot).